### DISCUSSION TITLE
Beta Coming Soon

### DISCUSSION THREAD

**Discussion by Ni1kko** [Ni1kko/HourBoostr#9] [General]
New updated version coming soon both apps merged into one whilst keeping the same look all code will be updated and all open issues with previous version will be fixed. Do keep in mind that the new app won't have every feature that the old one has straight away hence it being a beta but i want yous to be able too experience the new app before its finished. i will also be implementing any good idea the get enough up-votes

**Comment by kadewu1** [comment]
Thats so cool, waiting for updates.

**Comment by Ni1kko** [comment]
Little preview, Please note everything you see is subject to change so may not be how the finished product may look.


![a1](https://i.imgur.com/mDk75I2.png)
![a2](https://i.imgur.com/ShtShSd.png)
![a3](https://i.imgur.com/IQCHTAp.png)
![a4](https://i.imgur.com/wXIMtxB.png)
![a5](https://i.imgur.com/B24oVI7.png)
